app/Transformers/UserStatisticsTransformer.php

```diff
@@ -26,47 +26,86 @@
 
 class UserStatisticsTransformer extends Fractal\TransformerAbstract
 {
+    protected $availableIncludes = [
+        'rank',
+        'scoreRanks',
+        'user',
+    ];
+
     public function transform(UserStatistics\Model $stats = null)
     {
         if ($stats === null) {
             $stats = new UserStatistics\Osu();
         }
 
-        if ($stats->user_id !== null) {
-            $scoreRankClass = ScoreBest::class.'\\'.get_class_basename(get_class($stats));
-            $scoreRankCounts = $scoreRankClass::where('user_id', '=', $stats->user_id)
-                ->rankCounts()
-                [$stats->user_id] ?? null;
-        }
-
         return [
-            'rank' => [
-                'isRanked' => $stats->rank_score_index > 0,
-                'global' => $stats->rank_score_index,
-                'country' => $stats->countryRank(),
-            ],
-
             'level' => [
                 'current' => $stats->currentLevel(),
                 'progress' => $stats->currentLevelProgressPercent(),
             ],
 
             'pp' => $stats->rank_score,
-            'rankedScore' => $stats->ranked_score,
-            'hitAccuracy' => $stats->accuracy_new,
-            'playCount' => $stats->playcount,
-            'totalScore' => $stats->total_score,
-            'totalHits' => $stats->totalHits(),
-            'maximumCombo' => $stats->max_combo,
-            'replaysWatchedByOthers' => $stats->replay_popularity,
+            'pp_rank' => $stats->rank_score_index,
+            'ranked_score' => $stats->ranked_score,
+            'hit_accuracy' => $stats->accuracy_new,
+            'play_count' => $stats->playcount,
+            'total_score' => $stats->total_score,
+            'total_hits' => $stats->totalHits(),
+            'maximum_combo' => $stats->max_combo,
+            'replays_watched_by_others' => $stats->replay_popularity,
+            'grade_counts' => [
+                'ss' => $stats->x_rank_count,
```

Comment:
probably should be uppercase and X for ss just like other places? Then there's `beatmaps.rank.X` and stuff.